src/soc/sophgo/cv180x.c

```diff
@@ -24,6 +24,9 @@
 
 #define CV180X_GPIO_GROUP_COUNT 4
 
+// registers not open yet
+#define UNKNOWN_COMMON_VALUES 0, {0x0, 0x0}, {GPIO_SWPORTA_DDR, 0}, {GPIO_SWPORTA_DR, 0}, FUNCTION_UNKNOWN, PINMODE_NOT_SET, 0
```

Comment:
```suggestion
// register data currently unavailable
#define GPIO_UNAVAILABLE(x) ({x, 0, 0, {0, 0}, {0, 0}, {0, 0}, FUNCTION_UNKNOWN, 0, 0})
```
写宏的时候最好用括号包围，以免和其他部分的代码产生干扰
这里直接定义一个layout_t对象